### DISCUSSION TITLE
会支持suno 音乐生成吗

### DISCUSSION THREAD

**Discussion by keycaicurious** [yangjian102621/chatgpt-plus#209] [常规讨论]
会支持suno 音乐生成吗

**Comment by yangjian102621** [comment]
用中转 API，这个功能一直都支持的。

**Reply by yangjian102621** [reply]
![image](https://github.com/yangjian102621/geekai/assets/4746061/0a04e5ce-456f-489e-b439-13e0094ca1d6)
